### PR TITLE
use rcl_get_zero_initialized_client()

### PR DESCRIPTION
**Pull request by fujitatomoya**
Signed-off-by: Tomoya.Fujita <Tomoya.Fujita@sony.com>

### CODE REVIEW

**Comment by fujitatomoya** [comment]
@hidmic could you help me to merge this minor fix?

**Comment by fujitatomoya** [comment]
CI:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/13721/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/8593/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/11433/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/13794/)

File: rcl/test/rcl/test_client.cpp [hunk 1]
```diff
@@ -121,7 +121,7 @@ TEST_F(TestClientFixture, test_client_nominal) {
 TEST_F(TestClientFixture, test_client_init_fini) {
   rcl_ret_t ret;
   // Setup valid inputs.
-  rcl_client_t client;
+  rcl_client_t client = rcl_get_zero_initialized_client();
```
Comment (clalancette) [inline rcl/test/rcl/test_client.cpp:124]: Can you explain more about why we need this fix?  This is already done down below in line 137 (before client is ever used), so I don't see why this change is necessary.

Comment (fujitatomoya) [reply rcl/test/rcl/test_client.cpp:124]: ah, that is a true. maybe we don't need this line at all. i was just thinking that `rcl_get_zero_initialized_client` should be used to have initialized structure just in case.
